gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -572,6 +574,20 @@ case class TransformPreOverrides(isAdaptiveContext: Boolean)
         TransformHints.tagNotTransformable(newSource, validationResult.reason.get)
         newSource
       }
+    case plan if HiveTableScanExecTransformer.isHiveTableScan(plan) =>
+      // TODO: Add DynamicPartitionPruningHiveScanSuite.scala
```

Comment:
Is this suite disabled now?

Reply:
It's just introduced in spark 3.3. And may already be in our 3.3 UT plan list.